distribution/server-dist/src/main/docs/examples/map-storage-concurrenthashmap.cli

```diff
@@ -30,6 +32,6 @@ embed-server
 /subsystem=keycloak-server/spi=userSessions:add(default-provider=map)
 
 /subsystem=keycloak-server/spi=mapStorage:add(default-provider=concurrenthashmap)
-/subsystem=keycloak-server/spi=mapStorage/provider=concurrenthashmap:add(properties={dir="${jboss.server.data.dir}/map"},enabled=true)
+/subsystem=keycloak-server/spi=mapStorage/provider=concurrenthashmap:add(properties={dir="${jboss.server.data.dir}/map",keyType.realms=string,keyType.authz-resource-servers=string},enabled=true)
```

Comment:
Did I miss something or why we don't need `keyType.realms=string,keyType.authz-resource-servers=string` for undertow?

Reply:
there is this settings here https://github.com/keycloak/keycloak/blob/d80891637ca223635748dce4202880e860de209d/testsuite/integration-arquillian/tests/base/src/test/resources/META-INF/keycloak-server.json#L86-L87